blackmarket/UI/Views/SignInView.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  ContentView.swift
+//  blackmarket
+//
+//  Created by Daniel Parra on 9/27/22.
+//
+
+import SwiftUI
+
+struct SignInView: View {
+    
+    @ObservedObject var viewModel = SignInViewModel()
+    
+    var body: some View {
+        NavigationView {
+            VStack(spacing: 16.0) {
```

Comment:
We should keep consistency and use constant values from a UI enclosing enumeration.